src/registrar/config/settings.py

```diff
@@ -809,12 +810,22 @@ def format(self, record):
 SECRET_REGISTRY_KEY = secret_registry_key
 SECRET_REGISTRY_KEY_PASSPHRASE = secret_registry_key_passphrase
 SECRET_REGISTRY_HOSTNAME = secret_registry_hostname
+
+# endregion
+
+# region: DNS----------------------------------------------------------###
+
+# SECURITY WARNING: keep all DNS variables in production secret!
 SECRET_DNS_TENANT_KEY = secret_dns_tenant_key
 SECRET_DNS_TENANT_NAME = secret_dns_tenant_name
 SECRET_DNS_SERVICE_EMAIL = secret_registry_service_email
 SECRET_DNS_TENANT_ID = secret_dns_tenant_id
 
+# Configuration
+DNS_MOCK_EXTERNAL_APIS = dns_mock_external_apis
```

Comment:
I followed the existing pattern for env vars, but I wonder why we don't just initially set `DNS_MOCK_EXTERNAL_APIS` to the value of `dns_mock_external_apis`, so:
`DNS_MOCK_EXTERNAL_APIS = env.bool("DNS_MOCK_EXTERNAL_APIS", default=False)`

If you know or have context, lmk